phoenix-core/src/it/java/org/apache/phoenix/end2end/UpgradeIT.java

```diff
@@ -588,4 +587,87 @@ public void testMoveParentChildLinks() throws Exception {
         return childLinkSet;
     }
 
+    @Test
+    public void testMergeViewIndexSequences() throws Exception {
+        testMergeViewIndexSequencesHelper(false);
+    }
+
+    @Test
+    public void testMergeViewIndexSequencesWithNamespaces() throws Exception {
+        testMergeViewIndexSequencesHelper(true);
```

Comment:
This is really the only case where you need parameterization in this test suite, so just having the two tests with a common helper is easier (and quicker, since you don't get an extra test run from the other tests with a parameter it doesn't need.)